sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/VectorIndexType.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+/**
+ * Defines the index type of vector index specification in the Azure Cosmos DB service.
+ */
+public enum VectorIndexType {
+    /**
+     * Represents a flat vector index type.
+     */
+    FLAT("flat"),
+
+    /**
+     * Represents a quantized flat vector index type.
+     */
+    QUANTIZED_FLAT("quantizedFlat"),
+
+    /**
+     * Represents a disk ANN vector index type.
+     */
+    DISK_ANN("diskANN");
+
+
+    private final String overWireValue;
+
+    VectorIndexType(String overWireValue) {
+        this.overWireValue = overWireValue;
+    }
+
+    @Override
+    public String toString() {
+        return this.overWireValue;
+    }
+
+    /**
+     *
+     * @return value for the enum
+     */
+    public String getValue() {
```

Comment:
Same here.

Reply:
Retaining the getters as they are is essential for consistency. Therefore, even if getPath is not currently utilized, we should keep it here, as per Fabian's suggestion.